sidecar/cmd/main.go

```diff
@@ -40,8 +39,8 @@ import (
 )
 
 var (
-	scheme   = runtime.NewScheme()
-	setupLog = ctrl.Log.WithName("setup")
+	scheme = runtime.NewScheme()
+	logger = ctrl.Log.WithName("setup")
```

Comment:
When I was working on the code, I found `setupLog` a frustrating variable name for the package-level logger, so I renamed it